aggregator_core/src/datastore.rs

```diff
@@ -102,7 +102,7 @@ macro_rules! supported_schema_versions {
 // version is seen, [`Datastore::new`] fails.
 //
 // Note that the latest supported version must be first in the list.
-supported_schema_versions!(6, 5);
+supported_schema_versions!(7, 6, 5);
```

Comment:
If all our deployed Januses are up to schema version 6, I think we could drop 5 and claw back some CI time.